localisation/00_wc_sad.csv

```diff
@@ -73,34 +71,29 @@ learn_talon_form_desc;Learn how to shapeshift into a great bird.;;Lernt, Euch in
 learn_stag_form;Learn Stag Form;;Lernt die Hirschform;;;;;;;;;;;x
 learn_stag_form_desc;Learn how to shapeshift into a great stag.;;Lernt, Euch in einen Hirsch zu verwandeln.;;;;;;;;;;;x
 learn_master_cat_form;Master Cat Form;;Meistert die Katzenform;;;;;;;;;;;x
-learn_master_cat_form_desc;Attain mastery over the Cat Form.;;Erlangt Meisterschaft �ber die Katzenform.;;;;;;;;;;;x
+learn_master_cat_form_desc;Attain mastery over the cat form.;;Erlangt Meisterschaft �ber die Katzenform.;;;;;;;;;;;x
```

Comment:
I think druid forms should be upper case

Reply:
I prefer lower case. It's a description and the cat form isn't a proper noun.
If it refers to a gameplay feature, I would highlight it in yellow. It works too.

Reply:
it is a gameplay feature

Reply:
done